doc/api/stream.md

```diff
@@ -90,60 +97,224 @@ server.listen(1337);
 // error: Unexpected token o
 ```
 
-### Class: stream.Duplex
+[Writable][] streams (such as `res` in the example) expose methods such as
+`write()` and `end()` that are used to write data onto the stream.
 
-Duplex streams are streams that implement both the [Readable][] and
-[Writable][] interfaces.
+[Readable][] streams use the [`EventEmitter`] API for notifying application
+code when data is available to be read, or has been read, off the stream.
 
-Examples of Duplex streams include:
+[Duplex][] and [Transform][] streams are both [Writable][] and [Readable][].
 
-* [TCP sockets][]
+Applications that are either writing data to or consuming data from a stream
+are not required to implement the stream interfaces directly and will generally
+have no reason to call `require('stream')`.
+
+Developers wishing to implement new types of streams should refer to the
+section [API for Stream Implementors][].
+
+### Writable Streams
+
+Writable streams are an abstraction for a *destination* to which data is
+written.
+
+Examples of [Writable][] streams include:
+
+* [HTTP requests, on the client][]
+* [HTTP responses, on the server][]
+* [fs write streams][]
 * [zlib streams][zlib]
 * [crypto streams][crypto]
+* [TCP sockets][]
+* [child process stdin][]
+* [`process.stdout`][], [`process.stderr`][]
```

Comment:
We might note that some of these streams are `Duplex`, but that we'll only be focussing on their writable properties?